src/pages/ProfilePage.js

```diff
@@ -0,0 +1,150 @@
+                    styles.profilePageContainer,
+                ]}
+            >
+                <View>
+                    <View style={styles.settingsWrapper}>
+                        <View
+                            style={[styles.avatarLarge, styles.mb3]}
+                        >
+                            <Avatar
+                                style={[styles.avatarLarge]}
+                                source={profileDetails.avatarURL}
+                            />
+                        </View>
+                        <Text style={[styles.settingsDisplayName, styles.mt1, styles.mb6]} numberOfLines={1}>
+                            {profileDetails.displayName
+                                ? profileDetails.displayName
+                                : null}
+                        </Text>
+                        {profileDetails.login && (
+                        <View style={[styles.mb6, styles.profilePageSectionContainer]}>
+                            <Text style={[styles.profilePageLabel, styles.mb2]} numberOfLines={1}>
+                                {Str.isSMSLogin(profileDetails.login) ? 'Phone Number' : 'Email'}
+                            </Text>
+                            <Text style={[styles.profilePageLabel]} numberOfLines={1}>
+                                {Str.isSMSLogin(profileDetails.login)
+                                    ? Str.removeSMSDomain(profileDetails.login)
+                                    : profileDetails.login}
+                            </Text>
+                        </View>
+                        )}
+                        {profileDetails.pronouns && (
+                        <View style={[styles.mb6, styles.profilePageSectionContainer]}>
+                            <Text style={[styles.profilePageLabel, styles.mb2]} numberOfLines={1}>
+                                Preferred Pronouns
+                            </Text>
+                            <Text style={[styles.profilePageLabel]} numberOfLines={1}>
+                                {profileDetails.pronouns}
+                            </Text>
+                        </View>
+                        )}
+                        {profileDetails.timezone && (
+                        <View style={[styles.mb6, styles.profilePageSectionContainer]}>
+                            <Text style={[styles.profilePageLabel, styles.mb2]} numberOfLines={1}>
+                                Local Time
+                            </Text>
+                            <Text style={[styles.profilePageLabel]} numberOfLines={1}>
+                                {moment().tz(profileDetails.timezone).format('LT')}
+                            </Text>
+                        </View>
+                        )}
+                    </View>
+                </View>
+
+                <Text style={[styles.profilePageSectionVersion]} numberOfLines={1}>
+                    v
+                    {version}
+                </Text>
```

Comment:
I don't think we want the app version to exist on this screen.

Reply:
Ah, the issue mockups do show the app version on the screen, so I think this is fine actually. Though I agree it might be a little out of place to have them here.

Reply:
I agree, we only want the app version displayed on your own personal Settings page.

Reply:
@Maftalion - let's remove the app version from this page.